components/dashboard/src/teams/TeamBilling.tsx

```diff
@@ -34,6 +34,17 @@ export default function TeamBilling() {
     const [pendingTeamPlan, setPendingTeamPlan] = useState<PendingPlan | undefined>();
     const [pollTeamSubscriptionTimeout, setPollTeamSubscriptionTimeout] = useState<NodeJS.Timeout | undefined>();
 
+    console.log(
+        "members",
+        members.length,
+        "currency",
+        currency,
+        "teamSubscription",
+        teamSubscription,
+        "pendingTeamPlan",
+        pendingTeamPlan,
+    );
+
```

Comment:
**suggestion**: Another irrelevant-line-to-comment-on: Could we rename team plan names and drop the duplicate and unnecessary _Team_ prefix, following the specs?

From _Team Professional_ to  _Professional_, etc.

Reply:
I can easily hide the word "Team" in this page, if that's nicer visually, but we'll probably want to keep the word "Team" in all official team plans (e.g. to more easily differentiate them from individual plans in Chargebee, the DB, the Admin dashboard, etc)

Reply:
Any option could suffice for this iteration. I'll leave this to you to decide, @jankeromnes!

Reply:
I tried it, but then felt it looked too confusing -- using the same name as individual plans doesn't feel right. 🙈

So back to "Team" everywhere for now. Can be adjusted later, but also we'll deprecate these in favor of Usage-Based Billing in the not-too-distant future. 🔮

Reply:
Thanks for giving this a try, @jankeromnes! Going with plan :a: with the prefix sounds good.